.github/workflows/release-homebrew.yaml

```diff
@@ -0,0 +1,164 @@
+# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# When the installers are ready in installer private bucket, run this workflow to test them.
+# TODO: Add job of uploading installers to Github release when installer tests are stable.
+# "release" runner tag is to make target hosts deterministic for easy clean up.
+# TODO: Remove the "release" runner tag when installer tests are stable.
+name: Release Homebrew
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  get-latest-tag:
+    name: Get the latest release tag
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.latesttag.outputs.tag }}
+      version: ${{ steps.latestversion.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
+      - name: 'Get the latest tag'
+        id: latesttag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: v0.1.0
```

Comment:
Good point, will update it. Do you have any concern for using the third-party github action `WyriHaximus/github-action-get-previous-tag@v1`? I'm not clear about the policy for that.

Reply:
We use third-party actions in other workflows. Eg. `peter-evans/create-pull-request@v4` , so I think we should be good here. @ningziwen WDYT?

Reply:
Does the previous step refers to the release-installer workflow? We can trigger another workflow from workflow with personal access token of required permission or create a Github App. I think the former takes much less effort for implementation as long as we give right permission to the bot.

Reply:
If we want to avoid using `WyriHaximus/github-action-get-previous-tag@v1`, we could get the latest tag by checking out `finch` and running tag=\`git describe --tags $(git rev-list --tags --max-count=1)`.

Reply:
I think it's okay to use the action for now. What do you think?